tests/fixtures/ecma-version/6/modules/export-named-as-default.result.js

```diff
@@ -1,87 +1,179 @@
 module.exports = {
     "type": "Program",
+    "start": 0,
```

Comment:
I'm noticing these are new after updating these tests. It looks like Acorn has been adding the "start" and "end" properties to the `Program` node for a while.